File: AreaOfCircle/Area.ts
/*
    Author : Pakshal Jain
    Date : 28/10/2022
    Program : To Accept Radius From user and Calculate Area From user
*/

////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

//Function To Calculate Area of Circle
function Calculate(No : number) : number
{
    var area : number = 0;

    //Logic
    area = 3.14 * No * No;

    return area;
}

/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

//Main Function
function main() 
{
    var No : number = 5.3;
    var fret : number = 0.0;

    console.log("Jay Ganesh....");

    //Call To Calculate Function
    fret = Calculate(No);
    //Printing Area of Circle
    console.log("Area of Circle is : %f",fret);
}

//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

//Call To Main Function
main();

File: AreaOfCircle/Area.go
/*
	Author : Pakshal Jain
	Date : 28/10/2022
	Program : To Accept Radius from user and Print Area of Circle
*/

//Required Libraries
package main
import "fmt"

/////////////////////////////////////////////////////////////////////////////////////////////////////////////

//Function To Calculate Area of Circle
func Calculate(No float64) float64 {
	var ans float64 = 0.0;
	
	//Logic
	ans = 3.14 * No * No;

	return ans;
}

////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

//Main Function
func main() {
	var No float64 = 0.0;
	var fret float64 = 0.0;

	fmt.Println("Jay Ganesh...");

	//Taking Input From User
	fmt.Println("Enter Number....");
	fmt.Scanf("%f",&No);

	//Call To Calculate Function
	fret = Calculate(No);

	//Printing area of Circle
	fmt.Println("Area of Circle is : ",fret);
}